xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_imageenc_types.h

```diff
@@ -0,0 +1,82 @@
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+extern "C"
+{
+  struct IMAGEENC_INFO
+  {
+    int dummy;
+  };
+
+  struct IMAGEENC_PROPS
+  {
+    const char* mimetype;
+  };
+
+  struct ImageEncoder
```

Comment:
Can you change `struct ImageEncoder` to `typedef struct KodiToAddonFuncTable_ImageEncoder`.
Is to have a standardized start name of function tables on interface ([example](https://github.com/FernetMenta/kodi-agile/blob/master/xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h#L533). 

Related to the add-on system rework.